test/TEST-55-OOMD/test.sh

```diff
@@ -6,6 +6,18 @@ TEST_DESCRIPTION="systemd-oomd Memory Pressure Test"
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
 
+test_append_files() {
```

Comment:
You'll want to add `IMAGE_NAME="oomd"` above, so that you don't modify the shared image

Reply:
Hah, good catch, thanks!